content/200-orm/200-prisma-client/500-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -168,6 +168,7 @@ package:
     - 'node_modules/.prisma/client/libquery_engine-rhel-*'
     - '!node_modules/prisma/libquery_engine-*'
     - '!node_modules/@prisma/engines/**'
+    - '!node_modules/.cache/prisma/**'
```

Comment:
```suggestion
    - '!node_modules/.cache/prisma/**'  # only required for Windows
```

Reply:
@Jolg42 I don't have write access on this fork. Do you?

Reply:
@jharrell  No, I guess you could merge and then do a new PR